source/wp-content/themes/wporg-news-2021/block-templates/category-community.html

```diff
@@ -2,7 +2,7 @@
 <!-- This is a hack to ensure content isn't covered by the fixed header -->
 <!-- wp:template-part {"slug":"header","align":"full","className":"site-header-offset"} /-->
 
-<!-- wp:query {"tagName":"main","className":"site-content-container","query":{"category":"community","perPage":"20","postType":"post","inherit":false},"displayLayout":{"type":"flex","columns":4},"align":"full"} -->
+<!-- wp:query {"tagName":"main","className":"site-content-container","query":{"inherit":true},"displayLayout":{"type":"flex","columns":4},"align":"full"} -->
```

Comment:
```suggestion
<!-- Note: This query has filters, but they live in `category_posts_per_page()`. That's necessary to avoid a Gutenberg design flaw. -->
<!-- wp:query {"tagName":"main","className":"site-content-container","query":{"inherit":true},"displayLayout":{"type":"flex","columns":4},"align":"full"} -->
```

Reply:
"limitation" is probably a more generous word than "flaw" in my suggestion, especially since it's a beta feature. it'd probably also be good to link to https://github.com/WordPress/gutenberg/issues/32531, so that it's clear what the limitation refers to